### PR TITLE
Fix training service bugs

### PR DESCRIPTION
**Pull request by liuzhe-lz**
#### Test Options ####
  - [ ] fast test
  - [ ] full test - HPO
  - [ ] full test - NAS
  - [ ] full test - compression

### Checklist ###
  - [ ] test case
  - [ ] doc

### CODE REVIEW

File: ts/nni_manager/core/sqlDatabase.ts [hunk 1]
```diff
@@ -20,7 +20,15 @@ import { TrialJobDetail } from '../common/trainingService';
...
-create table TrialJobEvent (timestamp integer, trialJobId text, event text, data text, logPath text, sequenceId integer, message text);
+create table TrialJobEvent (
+    timestamp integer,
+    trialJobId text,
+    event text,
+    data text,
+    logPath text,
+    sequenceId integer,
+    message text,
+    environmentId text);
```
Comment (QuanluZhang) [inline ts/nni_manager/core/sqlDatabase.ts:31]: you changed the table which means the experiment created in previous version nni cannot be resumed by 3.0, right?

**Review by ultmaster — APPROVED** [review]

File: ts/nni_manager/core/sqlDatabase.ts (hunk 1 above)
Comment (liuzhe-lz) [reply ts/nni_manager/core/sqlDatabase.ts:31]: yes

Comment (QuanluZhang) [reply ts/nni_manager/core/sqlDatabase.ts:31]: then you should give this notice in a proper place

Comment (cruiseliu) [reply ts/nni_manager/core/sqlDatabase.ts:31]: I think it's enough in release note?

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**